__device-tests__/gutenberg-editor-sanity-test-4-visual.test.js

```diff
@@ -0,0 +1,235 @@
+/**
+ * Internal dependencies
+ */
+const { blockNames } = editorPage;
+const { toggleOrientation, selectTextFromTextInput, setClipboard } = e2eUtils;
+import { takeScreenshot } from './utils';
+
+describe( 'Gutenberg Editor - Test Suite 4', () => {
```

Comment:
I wonder if we should eventually stop grouping the suite numbers and organize them by subject matter instead, e.g. by block type. I presume the suite numbers were originally used to delegate relatively balanced batches of tests. 

WDYT?

Reply:
>I presume the suite numbers were originally used to delegate relatively balanced batches of tests.

Yeah, I think that was the main idea back then.

Currently, on CI, each test file takes quite a bit to get the environment ready to run the tests, I'd say around 2 minutes per file (on iOS at least) since the machines we use are not that powerful. I think we should avoid having too many files for now so the running time doesn't go too high.